src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -159,7 +159,7 @@ OP_ERROR SOP_SceneCacheSource::cookMySop( OP_Context &context )
 	{
 		sceneChanged();
 	}
-	
+	gdp->stashAll();
```

Comment:
I removed this, it clears the gdp, but keeps a list of the vertices around so that if similar geometry is recreated, then it is faster to recreate. It doesn't do anything in this case though.